File: examples/eApi v1.9/java/eAPI-v19-connector/src/main/java/cz/monetplus/mips/eapi/v19/connector/entity/responses/EchoCustomerResponse.java
package cz.monetplus.mips.eapi.v19.connector.entity.responses;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import cz.monetplus.mips.eapi.v19.connector.entity.SignBase;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;


@Data
@EqualsAndHashCode(callSuper = false)
@NoArgsConstructor @AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class EchoCustomerResponse extends SignBase {
    private String customerId;
    private int resultCode;
    private String resultMessage;

    @Override
    public String toSign() {
        StringBuilder sb = new StringBuilder();
        add(sb, customerId);
        add(sb, dttm);
        add(sb, resultCode);
        add(sb, resultMessage);
        deleteLast(sb);
        return sb.toString();
    }

}

File: examples/eApi v1.9/dotNET/CsobGTWDemo/CsobGatewayClientExample/Communication/DataObjects/Response/EchoCustomerResponse.cs
using System.Text;

namespace CsobGatewayClientExample.Communication.DataObjects.Response;

public class EchoCustomerResponse : SignBase
{
    public string? CustomerId { get; set; }
    public long ResultCode { get; set; }
    public string? ResultMessage { get; set; }

    public override string ToSign()
    {
        var sb = new StringBuilder();
        Add(sb, CustomerId);
        Add(sb, Dttm);
        Add(sb, ResultCode);
        Add(sb, ResultMessage);
        DeleteLast(sb);
        return sb.ToString();
    }
}